net/adblock/files/adblock-helper.sh

```diff
@@ -162,8 +182,23 @@ f_envcheck()
     network_find_wan6 adb_wanif6
     if [ -z "${adb_wanif4}" ] && [ -z "${adb_wanif6}" ]
     then
-        adb_wanif4="true"
-        f_log "no valid IPv4/IPv6 interface with default route found, IPv4 mode will be assumed"
+        adb_wanif4="${adb_lanif}"
+        f_log "no valid IPv4/IPv6 interface with default route found, IPv4/AP mode will be assumed"
+    fi
+    
+    # check AP mode
+    #
+    if [ "${adb_wanif4}" = "${adb_lanif}" ] || [ "${adb_wanif6}" = "${adb_lanif}" ]
+    then
+        adb_nullipv4="${adb_ipv4}"
+        adb_nullipv6="${adb_ipv6}"
+        if [ "$(uci get uhttpd.main.listen_http | grep -Fo "80")" = "80" ] ||
+           [ "$(uci get uhttpd.main.listen_https | grep -Fo "443")" = "443" ]
+        then
+            f_log "AP mode enabled, set local LuCI instance to ports <> 80/443"
```

Comment:
Do I understand correctly that in the new AP mode router's Luci interface will be inaccessible after the launch of adblock if the user does not first change uhttpd ports for Luci? And just a log message is filed into the system log...

If that is so, I would prefer adblock to stop its init and log an error about the needed port change, instead of launching and locking the user out of Luci...

There many users only using Luci, so the proposed change in this PR might lead into hard-to-debug complaints.